include/boost/url/params_view.hpp

```diff
@@ -43,6 +43,18 @@ class params_view
 public:
     class iterator;
 
+    /** The value_type of query parameters
```

Comment:
If we are going to accept this brief then it should say "The value type of query parameters"